openedx/xblock-drag-and-drop-v2
# feat: Add HTML support in explanation

## Description
This PR adds HTML support in explanations within DnDv2

After:
![screenshot-rendered-html](https://user-images.githubusercontent.com/554032/192121018-07a22a68-0efa-4994-938c-81aafdf36b6b.png)


Supporting information
OpenCraft internal ticket : [BB-6630](https://tasks.opencraft.com/browse/BB-6630)

## Testing instructions

1. Checkout this branch in <devstack_parent>/src directory in your local setup
2. log in to Studio shell, switch to edxapp virtual env
3. Go to /edx/src/xblock-drag-and-drop-v2 and run ```pip install -e .```
5. [enable the Drag and Drop XBlock in Studio](https://github.com/open-craft/xblock-drag-and-drop-v2/#enabling-in-studio)
5. Go to Studio UI and create a new unit
6. Select Problem > Advanced > Drag and Drop, to create a v2 drag and drop XBlock
7. Click Edit and change mode to Assessment
8. Scroll down to Explanation Text like `Test HTML 12m<sup>2</sup>`
9. Add some text in the text area
10. Click Continue twice and Save
11. Click on View Live Version
12. Ensure there is no extra line as shown above
13. Do as many attempts as required till "Show Answer" button is active
14. Click on Show Answer
15. Ensure Explanation is rendered like 'Test HTML  12m<sup>2</sup>'

## Deadline
"None"
